### PR TITLE
update monitoring page

### LINKED ISSUE #81: Link to Grafana dashboard and remove existing content
Remove the Creating a Grafana dashboard content.

See this discussion: https://consensys.slack.com/archives/GCEHWHAQ2/p1620094497068300

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull Request Description

Update monitoring page to remove "Creating a Grafana Dashboard" content and replace with link to GoQuorum dashboard. Also edit the rest of the page.

## Fixed Issue(s)

fixes #81 

## Preview

see https://pegasys-tessera--108.com.readthedocs.build/en/108/HowTo/Use/Monitoring/

### CODE REVIEW

File: docs/HowTo/Use/Monitoring.md [hunk 1]
```diff
@@ -134,20 +154,28 @@ To allow Tessera to communicate with a TLS-secured InfluxDB, `sslConfig` must be
...
+to get Prometheus setup and ready to integrate with Tessera.
+The [Prometheus First Steps](https://prometheus.io/docs/introduction/first_steps/) is a good starting point.
+A summary of the steps to store Tessera metrics in a Prometheus database are as follows:
 
-1. Install Prometheus
-1. Create a `prometheus.yml` configuration file to provide Prometheus with the necessary information to pull metrics from Tessera. A simple Prometheus configuration for use with the [7nodes example network](../../../../Getting Started/Quorum-Examples) is:
+1. Install Prometheus.
+1. Create a `prometheus.yml` configuration file to provide Prometheus with the necessary information to pull metrics
+   from Tessera.
+   A simple Prometheus configuration for use with the [7nodes example network](../../../../Getting Started/Quorum-Examples) is:
```
Comment (joshuafernandes) [inline docs/HowTo/Use/Monitoring.md:178]: I think we should replace all refs to the 7nodes example with the quickstart maybe? We can give them a direct link to example config too https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/goquorum/config/prometheus/prometheus.yml and a grafana dashboard working

File: docs/HowTo/Use/Monitoring.md [hunk 2]
```diff
@@ -4,50 +4,61 @@ description: Monitor Tessera
 
 # Monitor Tessera
 
-Tessera can be used with InfluxDB and Prometheus time-series databases to record API usage metrics. The data recorded can be visualised either by creating a custom dashboard or by using an existing dashboard tool such as Grafana.
+Tessera can be used with InfluxDB and Prometheus time-series databases to record API usage metrics.
```
Comment (joshuafernandes) [inline docs/HowTo/Use/Monitoring.md:7]: Might be replace 'and' with 'or'. It used to be that we needed both, but now either works and prom & grafana are our standard and we're consistent

File: docs/HowTo/Use/Monitoring.md [hunk 3]
```diff
@@ -4,50 +4,61 @@ description: Monitor Tessera
...
+The data recorded can be visualized by using an existing dashboard tool such as Grafana.
 
-In addition, Tessera logs can be searched, analyzed and monitored using Splunk. Splunk can be set up in such a way that the logs for multiple Tessera nodes in a network are accessible from a single centralized Splunk instance.
+In addition, Tessera logs can be searched, analyzed, and monitored using Splunk.
```
Comment (joshuafernandes) [inline docs/HowTo/Use/Monitoring.md:10]: Logs can be done via Splunk or ELK - we've got an example of Tessera logs in ELK in the quickstart too. Ref links: https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/common/logstash/pipeline/20_tessera.conf and https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/common/filebeat/filebeat.yml

**Review by joshuafernandes — APPROVED** [review]
Looks good Alexandra, thank you for this

File: docs/HowTo/Use/Monitoring.md [hunk 4]
```diff
@@ -4,50 +4,61 @@ description: Monitor Tessera
...
 Tessera can record the following usage metrics for each endpoint of its API:
 
-* Average Response Time
-* Max Response Time
-* Min Response Time
-* Request Count
-* Requests Per Second
+* Average response time
+* Maximum response time
+* Minimum Response Time
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:20]:
```suggestion
* Minimum response time
```

File: docs/HowTo/Use/Monitoring.md [hunk 5]
```diff
@@ -4,50 +4,61 @@ description: Monitor Tessera
...
 
-Both databases integrate well with the open source dashboard editor [Grafana](https://grafana.com/) to allow for easy creation of dashboards to visualise the data being captured from Tessera.
+Both databases integrate well with the open source dashboard editor [Grafana](https://grafana.com/) to allow for easy
+creation of dashboards to visualize the data being captured from Tessera.
 
 ### Using InfluxDB
 
-See the [InfluxDB documentation](https://docs.influxdata.com/influxdb) for details on how to set up an InfluxDB database ready for use with Tessera. A summary of the steps is as follows:
+See the [InfluxDB documentation](https://docs.influxdata.com/influxdb) for details on how to set up an InfluxDB database
+ready for use with Tessera.
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:39]:
```suggestion
The [InfluxDB documentation](https://docs.influxdata.com/influxdb) provides details on how to set up an InfluxDB database
for use with Tessera.
```

File: docs/HowTo/Use/Monitoring.md [hunk 6]
```diff
@@ -57,11 +68,16 @@ See the [InfluxDB documentation](https://docs.influxdata.com/influxdb) for detai
...
-    The InfluxDB HTTP API can be called directly as an alternative to using the `influx` CLI
 
-Each Tessera server type (for example `P2P`, `Q2T`, `ADMIN`, `THIRDPARTY`, `ENCLAVE`) can be configured to store API metrics in an InfluxDB. These servers can be configured to store metrics to the same DB or separate ones. Not all servers need to be configured to store metrics.
+    You can call the InfluxDB HTTP API directly as an alternative to using the `influx` CLI.
 
-To configure a server to use an InfluxDB, add `influxConfig` to the server configuration. For example:
+You can configure each Tessera server type (for example, `P2P`, `Q2T`, `ADMIN`, `THIRDPARTY`, `ENCLAVE`) to store API
+metrics in an InfluxDB.
+You can configure these servers to store metrics to the same database or separate ones.
+Not all servers need to be configured to store metrics.
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:77]: Existing issue, but I'm not sure what this means. Does it mean that we can optionally configure each Tessera server type (`P2P`, `Q2T`, `ADMIN`, `THIRDPARTY`, `ENCLAVE`)?

If so, maybe remove this line and update "You can configure each.." with "You can optionally configure each..."

File: docs/HowTo/Use/Monitoring.md [hunk 7]
```diff
@@ -134,119 +154,53 @@ To allow Tessera to communicate with a TLS-secured InfluxDB, `sslConfig` must be
 }
 ```
 
-where `truststore.jks` is a Java KeyStore format file containing the trusted certificates for the Tessera client (for example the certificate of the CA used to create the InfluxDB certificate).
+where `truststore.jks` is a Java KeyStore format file containing the trusted certificates for the Tessera client (for
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:157]: I think Besu uses "keystore".

File: docs/HowTo/Use/Monitoring.md [hunk 8]
```diff
@@ -134,119 +154,53 @@ To allow Tessera to communicate with a TLS-secured InfluxDB, `sslConfig` must be
...
+This isn't used so can simply be set as the trust store.
 
 ### Using Prometheus
 
-The [Prometheus documentation](https://prometheus.io/docs/introduction/overview/) provides all the information needed to get Prometheus setup and ready to integrate with Tessera. The [Prometheus First Steps](https://prometheus.io/docs/introduction/first_steps/) is a good starting point. A summary of the steps to store Tessera metrics in a Prometheus DB are as follows:
-
-1. Install Prometheus
-1. Create a `prometheus.yml` configuration file to provide Prometheus with the necessary information to pull metrics from Tessera. A simple Prometheus configuration for use with the [7nodes example network](../../../../Getting Started/Quorum-Examples) is:
+The [Prometheus documentation](https://prometheus.io/docs/introduction/overview/) provides all the information you need
+to get Prometheus setup and ready to integrate with Tessera.
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:171]:
```suggestion
The [Prometheus documentation](https://prometheus.io/docs/introduction/overview/) provides information 
to set up Prometheus to integrate with Tessera.
```

File: docs/HowTo/Use/Monitoring.md [hunk 9]
```diff
@@ -134,119 +154,53 @@ To allow Tessera to communicate with a TLS-secured InfluxDB, `sslConfig` must be
...
-
-1. Start Tessera. As Tessera always exposes the `metrics` endpoint no additional configuration of Tessera is required
-1. Start Prometheus
+1. Install Prometheus.
+1. Create a `prometheus.yml` configuration file to provide Prometheus with the necessary information to pull metrics
+   from Tessera.
+   You can use [an example Prometheus configuration](https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/goquorum/config/prometheus/prometheus.yml)
+   with the [Quorum Developer Quickstart](../../Tutorials/Quorum-Dev-Quickstart.md).
+1. Start Tessera.
+   As Tessera always exposes the `metrics` endpoint, no additional configuration of Tessera is required.
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:181]:
```suggestion
   Tessera always exposes the `metrics` endpoint, so no additional configuration is required.
```

File: docs/HowTo/Use/Monitoring.md [hunk 10]
```diff
@@ -134,119 +154,53 @@ To allow Tessera to communicate with a TLS-secured InfluxDB, `sslConfig` must be
...
-    1. Construct the query as shown in the below image. This retrieves the data for the `sendraw` API from the InfluxDB, finds the sum of the `RequestRate` for this data and groups by `instance` (that is each Tessera node). `time($_interval)` automatically scales the graph resolution for the time range and graph width.
-
-    The POST `sendraw` API is used by Quorum whenever a private transaction is sent using the `eth_sendTransaction` or `personal_sendTransaction` API. This panel gives a good indication of the private transaction throughput in Quorum. Note that if the `sendraw` API is called by another process, the count will not be a true representation of Quorum traffic.
-
-## Monitoring a Tessera network with Splunk
-
-Splunk can be used to search, analyze and monitor the logs of Tessera nodes.
-
-To consolidate the logs from multiple Tessera nodes in a network requires setting up Splunk and Splunk Universal Forwarders. The following pages from the Splunk documentation are a good starting point for understanding how to achieve this:
+Consolidating the logs from multiple Tessera nodes in a network requires setting up Splunk and Splunk Universal Forwarders.
```
Comment (bgravenorst) [inline docs/HowTo/Use/Monitoring.md:202]:
```suggestion
To Consolidate logs from multiple Tessera nodes, set up Splunk and Splunk Universal Forwarders.
```

**Review by bgravenorst — APPROVED** [review]
Some feedback

**Pull request merged**